android/capacitor/src/main/java/com/getcapacitor/BridgeActivity.java

```diff
@@ -39,7 +47,7 @@ protected void init(Bundle savedInstanceState, List<Class<? extends Plugin>> plu
      * use {@link BridgeActivity#bridgeBuilder}.
      */
     @Deprecated
-    protected void init(Bundle savedInstanceState, List<Class<? extends Plugin>> plugins, JSONObject config) {
+    protected void init(Bundle savedInstanceState, List<Class<? extends Plugin>> plugins, CapConfig config) {
```

Comment:
This is a breaking change. Can you add back the other implementation that accepts JSON and deprecate it similar to how the other ones are?

Reply:
After discussing, the breaking change is acceptable since the use case for using this method included Embedded and edge cases. Will document in upgrade guide. Added some javadoc to highlight that an instance of CapConfig is used instead of a JSONObject.